Remotion/Core/Reflection.CodeGeneration.TypePipe/Core.Reflection.CodeGeneration.TypePipe.csproj

```diff
@@ -14,6 +14,7 @@
     <ProjectReference Include="..\Core\Core.Core.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Remotion.ThirdPartyDependency.DynamicLanguageRuntime" Version="1.1.0.10" />
```

Comment:
we only need it here. in the future, we add this directly with re-motion typepipe and can drop it here.